packages/js/onboarding/changelog/46647-fix-wcpay-banner-mobile-screen

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Fix styling issues with WooPayments banner on mobile version.
```

Comment:
```suggestion
Fix styling issues in the WooPayments banner on mobile-sized viewports.
```

Reply:
Your suggestion seems better, I've updated it!